types/evernote/index.d.ts

```diff
@@ -1,6319 +1,1855 @@
-// Type definitions for evernote v 1.25.8
+// Type definitions for evernote v2.0.5
```

Comment:
Could you enable the `dt-header` in the `tslint.json` file? The version should only be major and minor.

Reply:
Sure, I'll enable it. I just didn't even check `tslint.json`. I just notice than a lot of rules are turned off.
When running `dts-gen`, it creates a `tslint.json` with only:

```json
{ "extends": "dtslint/dt.json" }
```
Should I just leave `tslint.json` as above (deleting all those disabled rules)? And, of course, I'll fix any that can be fixed.